### ISSUE TITLE
No listr-list/card anymore

### ISSUE THREAD

**Issue by payne-chris-r** [ga-wdi-boston/ember-components#11]
@jrhorn424 had a `listr-list/card` (or `listr-list-card`) component, which is no longer in this lesson. Not sure if it should still be there or not.

**Comment by gaand** [comment]
I don't think it was this lesson, but rather ember-resources.  I could be wrong.
